Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -345,6 +346,15 @@ public Task getWorkflowTask() {
         return workFlowTask;
     }
 
+    /**
+     * Returns whether the optimistic export flag is set.
+     *
+     * @return true if the optimistic export flag is set; false otherwise
+     */
+    public boolean isOptimisticExportFlagSet() {
+        return optimisticExportFlagSet;
+    }
+
```

Comment:
Method can now be removed as not used. Access to class field is only inside the class by direct access.